### DISCUSSION TITLE
Command correction - install & configure documentation

### DISCUSSION THREAD

**Discussion by ankyit** [crossplane/crossplane#2650] [Q&A]
While going through the documentation, I found the command `watch kubectl get pkg` on the page [https://crossplane.io/docs/v1.4/getting-started/install-configure.html](https://crossplane.io/docs/v1.4/getting-started/install-configure.html), should it be `kubectl get pkg`

Note that I tried with `watch` and it didn't work, but without `watch` it worked 

Ref: [https://crossplane.io/docs/v1.4/getting-started/install-configure.html#install-configuration-package](https://crossplane.io/docs/v1.4/getting-started/install-configure.html#install-configuration-package)

**Comment by jbw976** [comment]
Hey there @ankyit! What is the error that you see when running the `watch kubectl get pkg` command? Does it happen consistently, or just the first time you tried it? Thanks!
